www/src/components/auth/AuthButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { useAuth0 } from '@auth0/auth0-react';
+
+type BaseButtonProps = {
+    text: string;
+    onClick: () => void;
+};
+
+const BaseButton: FC<BaseButtonProps> = (props: PropsWithChildren<BaseButtonProps>) => {
+    return <button onClick={props.onClick}>{props.text}</button>;
+};
+
+const LogoutButton = () => {
```

Comment:
I just noticed, for consistency should this be typed (or does typescript infer this?)
```suggestion
const LogoutButton: FC = () => {
```

Reply:
No its not inferred as a functional component, just a function that returns a JSX Element. I'll add this for consistency, thanks for catching this!

Reply:
Typed all components, let me know if I missed any 👍